tests/integration/baseImage_test.go

```diff
@@ -25,6 +25,10 @@ func TestBaseImage(t *testing.T) {
 	if len(eveBaseRef) == 0 {
```

Comment:
This file is a good example. We can let this change go through, but these tests in general should be in the eve repo, not here. When we want to change code for eve and its attendant tests, they should be in a single PR in a single repo.

Reply:
100% agreed @deitch ! In fact as I mentioned to @sadov yesterday -- we should freeze all new test development for now and focus on polishing the usability of the eden framework and test themselves. Fo that it makes sense to keep this repo monolithic for now to help with the speed of refactoring. However, once we are ready to start creating new tests -- I absolutely agree it would make sense to actually migrate them into the proper repos.

P.S. when I say "usability" it is things like this I have in mind https://github.com/lf-edge/eden/issues/10